src/java/org/apache/cassandra/utils/vint/VIntCoding.java

```diff
@@ -134,6 +135,11 @@ public static int getVInt32(ByteBuffer input, int readerIndex)
         return checkedCast(decodeZigZag64(getUnsignedVInt(input, readerIndex)));
     }
 
+    public static long getVInt(ByteBuffer input, int readerIndex)
```

Comment:
This method seems unused

Reply:
left a comment at the other one, I impl every version (added dead code) just to have them there... I needed one of the 4 cases so figured it would be best to have all 4 at once rather than leave to the next author